### ISSUE TITLE
SVG attachment location for accidentals

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#277]
In SVG renderings, accidentals are attached to notes when the note is not in a chord, but is attached to the chord if the note is in a chord:

``` xml
<g class="note" id="note-L5F1">
        <path d="M1382 2045 L1706 2045" style="stroke-width: 20;" /> 
        <rect x="1637" y="1415" width="20" height="608" rx="0" style="stroke-width: 0;" />
        <g class="accid" id="accid-000000014157931"/>
</g>                                                                    
<g class="chord" id="chord-L6F1">
        <rect x="2440" y="965" width="20" height="1328" rx="0" style="stroke-width: 0;" />
        <path d="M2185 2045 L2509 2045" style="stroke-width: 20;" />
        <path d="M2185 2225 L2509 2225" style="stroke-width: 20;" />
        <g class="note" id="note-L6F1S1"/>                      
        <g class="note" id="note-L6F1S2"/>
        <g class="note" id="note-L6F1S3"/>                      
        <g class="accid" id="accid-000000084151883"/>           
        <g class="accid" id="accid-000000204227186"/>
        <g class="accid" id="accid-000000009391630"/>                   
</g>                                                      
```

This is probably done since the horizontal positions of the accidentals are calculated in coordination with each other, but this arrangement causes problems when highlighting notes:

![screen shot 2016-08-20 at 9 16 09 am](https://cloud.githubusercontent.com/assets/3487289/17829673/c6885b50-66b6-11e6-8b61-3343940a9ef1.png)

When highlighting notes, the accidentals on single notes are colored, but not those of chords.

If accidentals are attached to chords due to layout processing, note that this is not important, as notes across layers must all be considered at the same time when calculating their horizontal positions:

![screen shot 2016-08-20 at 9 20 11 am](https://cloud.githubusercontent.com/assets/3487289/17829701/5738b69a-66b7-11e6-97c3-647e90bde13f.png)

CSS styling for examples:

``` css
g[id^="note-"] {
    fill: orange;
}
```

It would be useful for accidentals to be treated in the same manner for chords and single notes, which mainly means to move accidentals to the note level in chords.  Otherwise, single notes should be placed into a structure similar to a chord with the notehead and accidental sibling elements (but that is less preferable).

**Comment by craigsapp** [comment]
See commit https://github.com/rism-ch/verovio/commit/941154fb0136eef5bd1592ca52775d50b7fc71f9#commitcomment-20563233 for another example.

**Comment by lpugin** [comment]
Fixed with 86be11c3fe5d2d03d5677460821642e737a75a81

**Comment by craigsapp** [comment]
Looks good:

<img width="375" alt="screen shot 2017-01-24 at 15 05 03" src="https://cloud.githubusercontent.com/assets/3487289/22270866/86ae87dc-e246-11e6-9e49-9f8194482f76.png">

But notice the MEI data, which will now can only generate `note/accid` and cannot create `note@accid` (at least as far as I can see):

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-01-24T15:06:38" version="0.9.14-dev-4fd8628">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-000000023298785">
                        <staffGrp xml:id="m-000000157538120">
                            <staffDef xml:id="staffdef-000000110597815" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-000000182073361">
                        <measure xml:id="measure-L4" n="1" right="end">
                            <staff xml:id="staff-L4F1" n="1">
                                <layer xml:id="layer-L4F1" n="1">
                                    <note xml:id="note-L5F1" dur="4" oct="4" pname="c" color="orange">
                                        <accid xml:id="accid-000000008252579" accid="s" />
                                    </note>
                                    <chord xml:id="chord-L6F1" dur="4">
                                        <note xml:id="note-L6F1S1" oct="3" pname="g" color="orange">
                                            <accid xml:id="accid-000000087867647" accid="f" />
                                        </note>
                                        <note xml:id="note-L6F1S2" oct="4" pname="f" color="orange">
                                            <accid xml:id="accid-000000161460390" accid="s" />
                                        </note>
                                        <note xml:id="note-L6F1S3" oct="4" pname="a" color="orange">
                                            <accid xml:id="accid-000000055723147" accid="n" func="caution" />
                                        </note>
                                    </chord>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```
